src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Solution/CurrentProjectSelectionStep.cs

```diff
@@ -131,20 +153,20 @@ async Task<ProjectCommand> CreateProjectCommandAsync(IProject project)
             {
                 return new ProjectCommand(project, isProjectCompleted(context, project), await RunChecksAsync(project, token).ConfigureAwait(false));
             }
+        }
 
-            async Task<bool> RunChecksAsync(IProject project, CancellationToken token)
-            {
-                var success = true;
-
-                foreach (var check in _checks)
-                {
-                    Logger.LogTrace("Running readiness check {Id}", check.Id);
+        private async Task<bool> RunChecksAsync(IProject project, CancellationToken token)
```

Comment:
This method appears to be the same as previous. Am I missing something or did the line numbers just throw of the change detection?

Reply:
It's not a local method anymore (it used to live inside CreateProjectCommandAsync; now it's a normal private method since it's called from a couple different methods. I don't think the method itself has changed, though.